_docs/payout-details.md

```diff
@@ -398,6 +403,23 @@ For Senegalese cash pickup requests please use:
 
 {% include language-tabbar.html prefix="xof-cash-details" raw=data-raw %}
 
+For Wizall cash pickup requests please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "phone_number": "774044436", // local Senegalese format
+  "identity_card_id": "2231324232", // Mandatory for Wizall
+  "identity_card_type": "PP", // Mandatory for Wizall; Values: "PP": Passport, "NI": National ID
+  "cash_provider": "wizall" // Mandatory for Wizall
+}
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xof-cash-details" raw=data-raw %}
+
 Please note when sending `XOF::Cash` payments you should subscribe to the `recipient.pending` webhook, as that will broadcast the payment reference ID the customer need to use to obtain the funds. Example webhook response excerpt:
```

Comment:
Please update the section below as well as Wizall does not support us sending the payment reference, and also we won't send back the cash pickup code. This is only for Wari